support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPageActions.js

```diff
@@ -2,39 +2,52 @@
 
 // ----- Imports ----- //
 
-import { type PaperBillingPlan } from 'helpers/subscriptions';
 import { ProductPagePlanFormActionsFor } from 'components/productPage/productPagePlanForm/productPagePlanFormActions';
-import { type PaperDeliveryMethod } from 'helpers/subscriptions';
-import { paperSubsUrl } from 'helpers/routes';
+import { paperCheckoutUrl, paperSubsUrl } from 'helpers/routes';
 import { getPaperCheckout } from 'helpers/externalLinks';
 import { sendClickedEvent } from 'helpers/tracking/clickTracking';
 
 import { type State } from './paperSubscriptionLandingPageReducer';
+import type { PaperFulfilmentOptions } from 'helpers/productPrice/fulfilmentOptions';
+import { HomeDelivery } from 'helpers/productPrice/fulfilmentOptions';
+import { getQueryParameter } from 'helpers/url';
+import type { PaperProductOptions } from 'helpers/productPrice/productOptions';
+import { Everyday } from 'helpers/productPrice/productOptions';
 
 // ----- Types ----- //
-export type TabActions = { type: 'SET_TAB', tab: PaperDeliveryMethod }
+export type TabActions = { type: 'SET_TAB', tab: PaperFulfilmentOptions }
 
 
 // ----- Action Creators ----- //
 
-const { setPlan } = ProductPagePlanFormActionsFor<PaperBillingPlan>('Paper', 'Paper');
-const setTab = (tab: PaperDeliveryMethod): TabActions => {
+const { setPlan } = ProductPagePlanFormActionsFor<PaperProductOptions>('Paper', 'Paper');
+const setTab = (tab: PaperFulfilmentOptions): TabActions => {
   sendClickedEvent(`paper_subscription_landing_page-switch_tab-${tab}`)();
-  window.history.replaceState({}, null, paperSubsUrl(tab === 'delivery'));
+  window.history.replaceState({}, null, paperSubsUrl(tab === HomeDelivery));
   return { type: 'SET_TAB', tab };
 };
 
+const getCheckoutUrl = (state: State) => {
+  if (getQueryParameter('displayCheckout') === 'true') {
+    const product = state.page.plan.plan ? state.page.plan.plan : Everyday;
+    return paperCheckoutUrl(state.page.tab, product);
+  }
+
+  const { referrerAcquisitionData, abParticipations, optimizeExperiments } = state.common;
+  return state.page.plan.plan ? getPaperCheckout(
```

Comment:
Minor: now that we have two functions for getting a paper checkout url it might be a bit clearer to call this `getLegacyPaperCheckout` / `getSubsFrontendPaperCheckout` (or similar)

Reply:
Yes, because it's in the externalLinks.js file I thought that would probably make it clear but I think you're probably right.

Reply:
That's a fair point, if I was looking at this code in an IDE (instead of reviewing in GitHub) I would just click through so it's no big deal.